app/controllers/application_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
-  before_action :enable_rack_mini_profiler
+  before_action :rack_mini_profiler
```

Comment:
hmmm I wasn't sure, since it's not only toggling the thing, but I'm ok with it, I don't think there's a word for `toggle and maybe show it if needed` xD

Reply:
I had to refresh the page after it was toggled on

Reply:
hmmm I think something might be happening the very frist time it's toggled in development after the server was started, that's the only consistent way I can reproduce this:

- make sure RMP is disabled, no cookie
- start the server `rails s`
- open any page
- add `?enable_rack_mini_profiler` and enter
- the widget doesn't show
- after this, disabling/enabling RMP shows it just fine every time without that extra refresh

I'm debugging this and the call to `authorize_request` is done correctly, for some reason it's not working that very first time, and I suspect something with development environment and lazy loading of classes? I don't think it's worth to do much more research, it seems to be something on the RMP gem